src/main/java/frc/robot/subsystems/AimingSubsystem.java

```diff
@@ -0,0 +1,244 @@
+
+    //initialize PID Controller Constants
+    mElevatorController.kP = AimingConstants.mElevatorPIDConstants.mKP;
+    mElevatorController.kI = AimingConstants.mElevatorPIDConstants.mKI;
+    mElevatorController.kD = AimingConstants.mElevatorPIDConstants.mKD;
+    mElevatorController.kS = AimingConstants.mElevatorPIDConstants.mKS;
+    mElevatorController.kV = AimingConstants.mElevatorPIDConstants.mKV;
+
+    mWristController.kP = AimingConstants.mWristPIDConstants.mKP;
+    mWristController.kI = AimingConstants.mWristPIDConstants.mKI;
+    mWristController.kD = AimingConstants.mWristPIDConstants.mKD;
+    mWristController.kS = AimingConstants.mElevatorPIDConstants.mKS;
+    mWristController.kV = AimingConstants.mElevatorPIDConstants.mKV;
+
+    mLeftElevatorMotor.getConfigurator().apply(mElevatorController);
+    mRightElevatorMotor.getConfigurator().apply(mElevatorController);
+    mWristMotor.getConfigurator().apply(mWristController);
+    //note: configuration uses internal encoders inside the motors, subject to change
+  
+    mLeftElevatorMotor.setInverted(true);
+  }
+
+  @Override
+  public void periodic() {
+    updateMotorModes();
+    elevatorPeriodic();
+    wristPeriodic();
+  }
+
+  private void elevatorPeriodic() {
+    //Clamping Rotation between domain
+    mDesiredElevatorDistanceIn = MathUtil.clamp(mDesiredElevatorDistanceIn, AimingConstants.MIN_ELEVATOR_DIST_IN, AimingConstants.MAX_ELEVATOR_DIST);
+
+    PositionVoltage elevatorVoltage = new PositionVoltage(getCurrentElevatorDistance()).withSlot(0);
+    mLeftElevatorMotor.setControl(elevatorVoltage.withPosition(getDesiredElevatorDistance()));
+    mRightElevatorMotor.setControl(elevatorVoltage.withPosition(getDesiredElevatorDistance()));
+
+    mCurrentElevatorDistanceIn = mLeftElevatorMotor.getRotorPosition().getValueAsDouble() * AimingConstants.ELEVATOR_ROTATIONS_TO_INCHES;
+  }
+
+  private void wristPeriodic() {
+
+    //Clamping Rotation between domain
+    mDesiredWristRotationDeg = MathUtil.clamp(mDesiredWristRotationDeg, AimingConstants.MIN_WRIST_ROTATION_DEG, AimingConstants.MAX_WRIST_ROTATION);
+  
+    PositionVoltage wristVoltage = new PositionVoltage(getDesiredWristRotation()).withSlot(0);
+    mWristMotor.setControl(wristVoltage.withPosition(getDesiredWristRotation()));
+    mCurrentWristRotationDeg = mWristMotor.getRotorPosition().getValueAsDouble() * 360;
+  }
+
+  private void updateMotorModes() {
+    SmartDashboard.updateValues();
+
+    AimingMotorMode mode = mChooser.getSelected();
+    
+    // Motors go towards setpoints
+    NeutralModeValue neutralmode = mode == AimingMotorMode.BRAKE ? NeutralModeValue.Brake : NeutralModeValue.Coast;
+
+    mLeftElevatorMotor.setNeutralMode(neutralmode);
+    mRightElevatorMotor.setNeutralMode(neutralmode);
+    mWristMotor.setNeutralMode(neutralmode);
+
+  }
+
+  // Contains Smart Dashboard Statements ONLY ON DEBUG
+  private void debugSmartDashboard() {
+
+    // TODO: Add debug statements
```

Comment:
You know what I'm going to say about TODO and GitHub issues :-)

Reply:
This issue has been resolved now.